client/packages/programs/src/JsonForms/components/DateOfBirth.tsx

```diff
@@ -80,9 +87,17 @@ const UIComponent = (props: ControlProps) => {
             value={dob ?? null}
             onChange={onChangeDoB}
             format="dd/MM/yyyy"
-            slotProps={{ textField: { sx: { width: 135 } } }}
+            width={135}
```

Comment:
Took me ages to work out why I couldn't get the error text to show up -- then I realised the DOB component was overwriting the whole "slotProps" object! (That's a lot of props to wipe out just to set a `width` 😆)